accounting/account/management/commands/generate_account_data.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2017-2018 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# pylint: disable=unused-variable,no-self-use
+"""
+Management command for generating data (usually for a development environment) to play with.
+"""
+
+import logging
+
+from django.core.management import BaseCommand
+from django.db import IntegrityError, transaction
+
+from accounting.account.tests import factories
+from accounting.bank.tests.factories import BankAccountFactory
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    A management command to generate data for the Account application.
```

Comment:
I'm still contemplating what to do with this management command: I think it'd be useful to move this to the `common` app, so that I can easily generate all types of data, including `BankAccount` data and `LineItem`/`Invoice` data. Right now since this is in `account` that doesn't make sense full sense to do.

I also made it only because I needed to easily generate data after having to drop my DB over and over for testing some stuff :p

Still, I will mention the detail. :+1: